### ISSUE TITLE
DEPRECATION NOTICE for ephfs-provisioner:v2.1.0-k8s1.11

### ISSUE THREAD

**Issue by saeed-mcu** [kubespray-offline/kubespray-offline#36]
Hi ,
The [Pushing and pulling with image manifest v2 schema 1](https://docs.docker.com/engine/deprecated/#pushing-and-pulling-with-image-manifest-v2-schema-1) Features will be removed from docker engine v27.0 and is disabled by default on v26.0

So if you try to pull image `cephfs-provisioner:v2.1.0-k8s1.11`
```
docker pull quay.io/external_storage/cephfs-provisioner:v2.1.0-k8s1.11
```
it will be failed.

**Comment by tmurakam** [comment]
Duplicate of #35

**Comment by tmurakam** [comment]
This was already fixed v2.26.0-0, and it uses podman to download the image.
